src/features/bookings/components/BookingSeatsCta.tsx

```diff
@@ -60,9 +65,14 @@ export const BookingSeatsCta = ({
               seatId: seatId ? Number(seatId) : undefined,
               slotId: Number(slotId),
             })
-            .then(() => {
+            .then(async () => {
               onCloseModal?.();
-              navigation.pop(2);
+              await navigation.pop(2);
+              setTimeout(() => {
+                AccessibilityInfo.announceForAccessibility(
+                  t('bookingSeatScreen.confirmSuccess'),
+                );
+              }, 2000);
```

Comment:
saw this kind of block several times already... maybe making a helper util can help